pkg/gen_testcase.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash
+
+getListing() {
+	_pkgdbdir="`pwd`/pkg.$$"
+	rm -rf "$_pkgdbdir"
+	mkdir "$_pkgdbdir"
+	cp "$1" "$_pkgdbdir/Packages"
+	rpm -qa --dbpath "$_pkgdbdir" --queryformat "\{%{EPOCH}, \"%{NAME}\", \"%{VERSION}\", \"%{RELEASE}\", \"%{ARCH}\", \"%{VENDOR}\", \`%{SUMMARY}\`, %{SIZE}, %{INSTALLTIME}\},\n" | sed "s/^{(none)/{0/g" | sed "s/(none)//g" | sed "s/^/		/g"
+	rm -rf "$_pkgdbdir"
+}
+
+cat << EOT
+package rpmdb
+
+var (
+	// docker run --rm -it centos:6 bash
+	// rpm -qa --queryformat "\{%{EPOCH}, \"%{NAME}\", \"%{VERSION}\", \"%{RELEASE}\", \"%{ARCH}\", \"%{VENDOR}\", \"%{SUMMARY}\", %{SIZE}, %{INSTALLTIME}\},\n" | sed "s/(none)/0/g"
+	CentOS6Plain = []PackageInfo{
+`getListing "Packages_centos6_plain"`
```

Comment:
How does it work? The file path doesn't exist.

Reply:
Sorry,I forgot to refine test cases,the second patch fixed it.

Reply:
@knqyf263 I've create a new [pr](https://github.com/knqyf263/go-rpmdb/pull/18).